testgrid/config.md

```diff
@@ -113,7 +113,7 @@ dashboard_groups:
 
 ## Testing your configuration
 
-Run `bazel test //testgrid/...` to ensure the config is valid.
+Run `bazel test //testgrid/...` to ensure the [configuration] is valid.
```

Comment:
really this is this is the testgrid.k8s.io configuration

Reply:
It may be neat to move the testgrid config validation tests to `config/tests/testgrids` at some point so that `bazel test //config/...` enforces all of our conventions; right now what we have in `//testgrid/...` is a combination of validation and convention enforcement

Reply:
Added an issue: #13690 
After writing it, I think I could just do this while I write the extra testing for Configurator: #13638